web/src/main/java/edu/softserve/zoo/web/security/LoginSuccessHandler.java

```diff
@@ -0,0 +1,22 @@
+package edu.softserve.zoo.web.security;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Simple login success handler. Used for overriding other success handlers where no redirect functionality is needed.
+ *
+ * @author Ilya Doroshenko
+ */
+@Component
+public class LoginSuccessHandler implements AuthenticationSuccessHandler {
```

Comment:
To override `SavedRequestAwareAuthenticationSuccessHandler` which is used by Spring by default in `AbstractAuthenticationProcessingFilter` and has extra redirect and request caching functionality we don't need. 
We need to return response to user immediately, not to redirect him anywhere.